components/post_view/post_body/post_body.jsx

```diff
@@ -225,8 +225,10 @@ export default class PostBody extends React.PureComponent {
             </div>
         );
 
+        const hasPlugin = post.type && this.props.pluginPostTypes.hasOwnProperty(post.type);
+
         let messageWithAdditionalContent;
-        if (this.props.post.state === Posts.POST_DELETED) {
+        if (this.props.post.state === Posts.POST_DELETED || hasPlugin) {
```

Comment:
Don't want to show link/image previews, youtube videos, etc. when the post is being overridden by a plugin component